### ISSUE TITLE
Media Bulkloader: Images Not Showing in Records

### ISSUE THREAD

**Issue by phrynosoma13** [ArctosDB/arctos#5664]
Hello! It's been two weeks since the media bulkloader was used to import 546 images to their corresponding records in Arctos (TMCC:Herb), and I don't see any images showing. @Jegelewicz was going to bump these images to the top of the wait list, and I'm not sure if 2+ weeks is a normal wait time or if there is an issue.

Here is the GitHub thread with all details pertaining to the TMCC:Herb collections, including the media bulkloading process: https://github.com/ArctosDB/data-migration/issues/1032

Thank you!
Megan

**Comment by Jegelewicz** [comment]
@phrynosoma13 your media did not load

![image](https://user-images.githubusercontent.com/5725767/219151914-c3692645-df3d-415b-ac00-7bbd35a5cd06.png)

https://arctos.database.museum/tools/BulkloadMedia.cfm#phrynosoma

I tried one of the urls - https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/1074 and it gets me a 404 not found.

Let me dig around to figure out what is wrong with those urls

**Comment by phrynosoma13** [comment]
@Jegelewicz Thanks- I wasn't sure if that was a general message until the images are posted or otherwise approved...

I think I see the issue- the '.jpg" is missing from the end of the urls!

Does this solve the problem?

-Megan

**Comment by Jegelewicz** [comment]
@dustymc I feel like the url is just being built incorrectly above, but I don't find any good documentation on how it should be built.

**Comment by Jegelewicz** [comment]
the '.jpg" is missing from the end of the urls!

that still doesn't work

https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/1074.jpg

gives 404

**Comment by phrynosoma13** [comment]
Shoot, OK... I'll leave this to the experts! :D

**Comment by Jegelewicz** [comment]
Only possible choices from https://web.corral.tacc.utexas.edu/ are 

![image](https://user-images.githubusercontent.com/5725767/219156990-3f4ae03d-df04-4607-bc87-c6279e1c6057.png)


So your things are somewhere else.

**Comment by dustymc** [comment]
https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/1074.JPG

**Comment by phrynosoma13** [comment]
This is where I put the photos, and photos show when the individual links are accessed: https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/

**Comment by Jegelewicz** [comment]
https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/1000.JPG

works so the .JPG has to be uppercase

@phrynosoma13 you should delete what is in the media bulkload tool, add the .JPG to your URIs and re-load. Sorry I didn't try one of those before you loaded them the first time!

**Comment by dustymc** [comment]
@phrynosoma13 that will also fail, made date has been excellified.

```
https://web.corral.tacc.utexas.edu/arctos/TMCC/Herb/2023_01_22/1000.JPG,image/jpeg,image,,CC BY-SA,description,whole specimen,created by agent,Megan Lahti,shows cataloged_item,TMCC:Herb:1000,made date,12/8/2021
```

>.JPG has to be uppercase

In general, it has to exactly match.

**Comment by phrynosoma13** [comment]
Shoot. I just uploaded the file to the media bulkloader and selected "autoload" before seeing this message regarding the "made date" formatting... The bulkloader status indicator didn't show any errors/issues in the preview window, and after I selected "autoload" the bulkloader status indicator now shows "autoloaded." How do I fix this?

And yes, I see that the ".JPG" is case sensitive- thanks!

Sorry! :(
Megan
